treeherder/seta/analyze_failures.py

```diff
@@ -59,8 +59,16 @@ def get_failures_fixed_by_commit():
     failures = {}
     # We're assuming that sheriffs always anotate failed jobs correctly using "fixed by commit"
     for job_note in models.JobNote.objects.filter(failure_classification=2):
```

Comment:
That adds the join to the jobs table, which is a start, but needs the others too. In the other bug (about slow API, but root cause is this same line) I gave Armen an example of how to fix it:
https://bugzilla.mozilla.org/show_bug.cgi?id=1330354#c5